docs/edgeql/ddl/migrations.rst

```diff
@@ -230,28 +230,33 @@ in the current migration block in the specified output format:
     .. code-block::
 
         {
-          // List of confirmed migration
-          // statements.
+          // Name of the parent migration
```

Comment:
(should be `.. code-block:: javascript`)

Reply:
I tried that, it errors, because it's not really valid JS.